conda_lock/src_parser/pyproject_toml.py

```diff
@@ -380,3 +342,41 @@ def parse_pdm_pyproject_toml(
     res.dependencies.extend(dev_reqs)
 
     return res
+
+
+def parse_pyproject_toml(
```

Comment:
Mainly for code-discoverability. The general pattern in most of the modules is that the last function in a module is the main function exposed by that module and used by the rest of the program. Since `parse_pyproject_toml` is the main exported function, I thought it should be at the end to fit that pattern.